build.rs

```diff
@@ -193,3 +195,27 @@ fn setup_paths() {
     rsconf::set_env_value("DOCDIR", docdir.to_str().unwrap());
     rsconf::rebuild_if_env_changed("DOCDIR");
 }
+
+fn get_version(build_dir: &str) -> String {
+    use std::fs::read_to_string;
+    use std::process::Command;
+
+    if let Ok(var) = std::env::var("FISH_BUILD_VERSION") {
+        return var;
+    }
+
+    let path = PathBuf::from(build_dir).join("version");
```

Comment:
I would like to (at some point) rename this file to `FISH_BUILD_VERSION` to make everything more consistent.

Reply:
Tbh my preference would be to pass the version as an environment variable - this would also be much easier for sphinx to read when building manpages (which would allow us to build them before building the binaries, which makes it possible to embed them in a binary).

I haven't figured out how to get cmake to do that, tho